docs/ldml/tr35-keyboards.md

```diff
@@ -2229,7 +2229,7 @@ _Attribute:_ `from` (required)
 
 - **Nested capturing groups**
 
-    `(?:[abc]([def]))|(?:[ghi])`
+    `(?:[abc]​([def]))|(?:[ghi])`
```

Comment:
```suggestion
    `(?:[abc]([def]))|(?:[ghi])`
```

This is a sample, people will want to copy and paste it.

Another approach you can take for the link checker is to check the output HTML - then check the resolved `href=`